TableWidgets.go

```diff
@@ -132,6 +134,12 @@ func (t *TableWidget) FastMode(b bool) *TableWidget {
 	return t
 }
 
+// NoHeader indicates that the column header should not be shown.
```

Comment:
Add here that "it allows to setup column flags in `Columns` method" and say something about this method in a comment over Columns method - just for documentation

Reply:
Okay. New change committed